### ISSUE TITLE
Scroll to entity using hyperlink

### ISSUE THREAD

**Issue by ishan-krishna** [NightCafeStudio/react-render-if-visible#4]
Hi,

Great library first of all!

I was wondering if it is possible to have an option to scroll to a child (with/without animation) on its corresponding hyperlink click? Suppose we have 5 entities sections `a,b,c,d,e` and out of them only `a` is visible on the screen. Now if I want to add a `table of contents` on my page and want to attach a hyperlink to `e` which will directly scroll to `e` section , how is it possible with this library?

Thanks for your help.

**Comment by yelnyaface** [comment]
Hi, can this plugin do what the OP said?

**Comment by GusRuss89** [comment]
Yes you can do this, just wrap the element with id that you want to scroll to around the `<RenderIfVisible />` component.

E.g.

```javascript
<div id="a">
  <RenderIfVisible>
    <MyComponent />
  </RenderIfVisible>
</div>
```

**Comment by yelnyaface** [comment]
I see, but if `<MyComponent />` height is dynamic and loaded with `async useEffect()` it will not scroll to the correct new vertical position right? because the `<div id="a">` content wont be loaded unless it was scrolled to before?
